### DISCUSSION TITLE
SysLog log level ALL is TRACE

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#313] [Q&A] [answered]
I remember this was for a reason:

![image](https://user-images.githubusercontent.com/1230712/150603421-732b9312-60a4-493f-a127-8f8a889cb30f.png)

@MichaelDvP is this ok or does it need looking into?

**Comment by MichaelDvP** [comment]
You have always skip trace-setting and name level 8 to `ALL`, level 8 and 9 have same output. In my dev2 i have used level 9 for `ALL`, make `Trace` selectable, modified level to 8 to output only telegrams. But this was a private test does not fit to the loglevel scheme.
I've always thought you use 8 only because it is the next enum.

**Comment by proddy** [comment]
to be honest I can't remember anymore. A lot of the code back then was either written at airports or after too many beers. I wanted the log to follow the [standards](https://sematext.com/blog/logging-levels/) but users we're missing ALL to show everything I didn't understand the cascade mechanism. I'd prefer to remove ALL and go back to TRACE so it's consistent with everything else and use whatever changes you made in your dev2.

**Reply by MichaelDvP** [reply]
No, `ALL` is a valid (meta) level, set the value [here](https://github.com/emsesp/EMS-ESP32/blob/a7df1a2ba71ecf85617c55ca99c8c699f83dd962/interface/src/project/SettingsApplication.tsx#L490) to 9 and it matches.
Check [here](https://github.com/emsesp/EMS-ESP32/blob/a7df1a2ba71ecf85617c55ca99c8c699f83dd962/lib/uuid-log/src/uuid/log.h#L89).

**Comment by proddy** [comment, marked as answer]
ok, so add ALL back to the WebUI for Syslog?

**Reply by MichaelDvP** [reply]
Only set the value={9} instead of 8 in the [menu](https://github.com/emsesp/EMS-ESP32/blob/a7df1a2ba71ecf85617c55ca99c8c699f83dd962/interface/src/project/SettingsApplication.tsx#L490).

**Reply by MichaelDvP** [reply]
And [here](https://github.com/emsesp/EMS-ESP32/blob/a7df1a2ba71ecf85617c55ca99c8c699f83dd962/interface/src/framework/system/SystemLog.tsx#L213)

**Reply by proddy** [reply]
thanks, done.
